string pointset
string meta_category
---
string[] name
string[] description
string[] goto_node
string[] node_type
